### PR TITLE
SAAS-1301 advisors

### PR DESCRIPTION
**Pull request by nicolalamacchia**

### CODE REVIEW

File: src/pages/Advisors/Advisors.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import { WelcomeModal } from 'components/WelcomeModal';
+import { getStyles } from './Advisors.styles';
+import { Messages } from './Advisors.messages';
+
+export const AdvisorsPage: FC = () => {
+  const styles = useStyles(getStyles);
+  const [isModalVisible, setIsModalVisible] = useState(true);
+
+  const handleModalToggle = () => {
+    setIsModalVisible(!isModalVisible);
```
Comment (doracretu3pillar) [inline src/pages/Advisors/Advisors.tsx:13]: Shouldn't the Welcome from Advisors behave the same as DBaaS Welcome banner by setting a boolean property in the localStorage ?

File: src/components/WelcomeModal/WelcomeModal.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        {Messages.start}
+      </Button>
+    </div>
+  );
+
+  return (
+    <>
+      <div className={styles.infoButton}>
+        <Button onClick={handleToggle} startIcon={<HelpOutlineIcon />} data-testid="welcome-more-info-button">
+          {' '}
```
Comment (doracretu3pillar) [inline src/components/WelcomeModal/WelcomeModal.tsx:30]: is {' '} necessary ?

File: src/components/WelcomeModal/WelcomeModal.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+          {Messages.moreInfo}
+        </Button>
+      </div>
+      <SimpleDialog
+        maxWidth="md"
+        title={title}
+        open={isModalVisible}
+        onClose={handleToggle}
+        actions={DialogActions}
+        classes={{ root: styles.root }}
```
Comment (doracretu3pillar) [inline src/components/WelcomeModal/WelcomeModal.tsx:40]: no need to use classes if we have one class

File: src/pages/Advisors/Advisors.tsx (hunk 1 above)
Comment (nicolalamacchia) [reply src/pages/Advisors/Advisors.tsx:13]: done

File: src/components/WelcomeModal/WelcomeModal.tsx (hunk 2 above)
Comment (nicolalamacchia) [reply src/components/WelcomeModal/WelcomeModal.tsx:30]: done

**Review by doracretu3pillar — APPROVED** [review]

**Pull request merged**